logicrunner/handle_calls.go

```diff
@@ -79,14 +79,17 @@ func (h *HandleCall) handleActual(
 	}
 	es.Unlock()
 
+	// RegisterRequest is an external, slow call to the LME thus we have to
+	// unlock ExecutionState during the call.
+
 	procRegisterRequest := NewRegisterRequest(parcel, h.dep)
 
 	if err := f.Procedure(ctx, procRegisterRequest, true); err != nil {
 		if err == flow.ErrCancelled {
 			// Requests need to be deduplicated. For now in case of ErrCancelled we may have 2 registered requests
 			return nil, err // message bus will retry on the calling side in ContractRequester
 		}
-		return nil, os.WrapError(err, "[ Execute ] can't create request")
+		return nil, os.WrapError(err, "[ HandleCall ] can't create request")
```

Comment:
it's method handleActual but error message contains HandleCall, seems strange

Reply:
ok fixed